pkg/scheduler/util/predicate_helper.go

```diff
@@ -71,7 +71,7 @@ func (ph *predicateHelper) PredicateNodes(task *api.TaskInfo, nodes []*api.NodeI
 
 		// TODO (k82cn): Enable eCache for performance improvement.
 		if _, err := fn(task, node); err != nil {
-			klog.V(3).Infof("Predicates failed: %v", err)
+			klog.V(5).Infof("Predicates failed: %v", err)
```

Comment:
what if user want to know the specific err message of each node?

Reply:
It is on the same order of magnitude as failed node count * pod count.

Reply:
Thank you for the correction. So are you agree to modify the log level?

The current scheduler does not have a delay queue, and failed tasks will be retried immediately.